client/src/components/VideoCard.jsx

```diff
@@ -3,25 +3,25 @@ import { useState } from "react";
 import { Container } from "react-bootstrap";
 
 function VideoCard({ rating, title, url }) {
-  const [vote, setvote] = useState(rating);
+  const [like, setLike] = useState(rating);
 
+  // adds one vote
   const handleVoteUp = (e) => {
-    let value = e.target.value;
     e.preventDefault();
-    console.log(value);
-    setvote(rating + 1); // this is to add only one vote per user, another word i don't want user to keep adding more votes.
+    setLike(rating + 1);
   };
 
+  // setting back to the original rating number, and can substract one
   const handleVoteDown = (e) => {
     e.preventDefault();
-    setvote(rating - 1); // this is to add only one vote per user, another word i don't want user to keep adding more votes.
+    like > rating ? setLike(rating) : setLike(rating - 1);
```

Comment:
Good work. Would make this compare against the numbers instead of doing it in this roundabout way. e.g. if rating == 0 don't take any action (though some sites like StackOverflow let you have minus ratings, so that's valid also)